src/test/url-handling.test.js

```diff
@@ -103,7 +103,7 @@ describe('threadOrder and hiddenThreads', function() {
 
   it('can reorder the threads ', function() {
     const { getState } = _getStoreWithURL(
-      { search: '?threadOrder=1-2-0' },
+      { search: '?threadOrder=1-2-0&thread=0' },
```

Comment:
A selected thread is required now in many of these tests, as it will assume that the `VIEW_PROFILE` action has not been fired. No URL upgrader should be needed, as the tests are the only that are affected.

Reply:
Ah, that's much better, thanks.